moderator.lua

```diff
@@ -209,6 +209,29 @@ function HandleTPSCommand(Split, Player)
 	return true
 end
 
-
-
-
+function HandleSkullCommand(Split, Player)                                                                                                                              
+	if Split[2] == nil then                                                                                                                                         
+		Player:SendMessageInfo("Usage: "..Split[1].." <username>")                                                                                            
+		return true                                                                                                                                             
+	end                                                                                                                                                             
+                                                                                                                                                                        
+	pos = GetPlayerLookPos(Player)                                                                                                                                  
```

Comment:
Temporarily used variables should always be local, otherwise you get a memory overflow